src/components/DashboardHome/Sidebar.jsx

```diff
@@ -2,10 +2,27 @@ import React from 'react';
 
 import Links from '../Links/Links';
 import linksData from './sampleLinks';
+import SidebarBlock from './SidebarBlock';
 
 const Sidebar = () => (
   <>
-    <Links id={linksData.id} title={linksData.title} links={linksData.links} />
+    <SidebarBlock title="Program Document" sectionClassName="links">
+      <Links id={linksData.id} links={linksData.links} />
+    </SidebarBlock>
+    <SidebarBlock title="Manage Your Degree">
+      <p>This could be any bunch of children passed in somehow.</p>
+      <ul>
+        <li>Thing one</li>
+        <li>Thing two</li>
+      </ul>
+    </SidebarBlock>
+    <SidebarBlock title="Get Technical Support">
```

Comment:
Just to match what's in the mocks, let's have the children of this `SidebarBlock` include the external link at the bottom with the appropriate `FontAwesomeIcon` (https://fontawesome.com/icons/external-link-alt?style=solid):

![image](https://user-images.githubusercontent.com/2828721/58127313-10228080-7be3-11e9-9d82-34cd4ead05ff.png)